tools/pg_tools/pg-query.xml

```diff
@@ -0,0 +1,48 @@
+<tool id="pg_query" name="Postgresql" version="@PG_VERSION@">
+    <description>import sql dump</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+            @UNTAR_INFILE@ &&
+            @PG_START@ &&
+            ls -l ./postgresql
+            &&
+            @PSQL@ -c '$query' -L logfile.log > '$outfile' &&
```

Comment:
could we wrap this in a `copy ($query) to stdout with csv delimiter '	';` (i.e. raw tab char)? Then you'd get automatic tabular files. (or at least make it an option and make tab output default)